RecoParticleFlow/PFClusterProducer/python/particleFlowRealisticSimClusterHGCCalibrations_cfi.py

```diff
@@ -2,5 +2,5 @@
 
 minEtaCorrection = cms.double(1.4)
 maxEtaCorrection = cms.double(3.0)
-hadronCorrections = cms.vdouble(1.16, 1.10, 1.08, 1.08, 1.09, 1.11, 1.14, 1.26)  
-egammaCorrections = cms.vdouble(1.00, 1.00, 1.00, 1.00, 1.02, 1.03, 1.03, 1.03)  
+hadronCorrections = cms.vdouble(1.24, 1.24, 1.24, 1.23, 1.24, 1.25, 1.29, 1.29)  
```

Comment:
is it clear how the supposedly minor effect of the fix is affecting the hadron correction so significantly.
Some eta bins are up by 15% (compared to at most 1% for egamma corrs).

Reply:
the energy of the hits outside the radius is not accounted anymore, so it's not a minor effect for hadrons.
clusters from 50GeV pions have usually a radius of 10-ish cm (in FH), and are much smaller (2-3cm) in EE.
One could probably set two radii in the future depending on the subdetector...